app/lib/actions/candlepin/consumer/remove_subscription.rb

```diff
@@ -7,11 +7,17 @@ class RemoveSubscription < Candlepin::Abstract
         input_format do
           param :uuid, String
           param :entitlement_id, String
+          param :pool_id, String
         end
 
         def run
           ::Katello::Resources::Candlepin::Consumer.remove_entitlement(input[:uuid], input[:entitlement_id])
         end
+
+        def finalize
+          pool = ::Katello::Pool.where(:cp_id => input[:pool_id]).first
+          pool.import_data if pool
```

Comment:
can a test be added in `test/actions/katello/host/remove_subscriptions_test.rb` for this? It might be a hassle to test the `finalize`, I'm not sure if it would be worth it or not.